acsys/__init__.py

```diff
@@ -694,6 +694,15 @@ async def request_stream(self, remtsk, message, *, proto=None, timeout=1000):
 isn't an integer, ValueError is raised.
 
         """
+
+        pending = set([])
+
+        # If the caller provides a "signalling" future, wrap it in a
+        # `task` (required by `asyncio.wait()`.)
+
+        if done_fut is not None:
+            pending.add(asyncio.create_task(done_fut))
```

Comment:
`.create_task` takes a coroutine. `done_fut` is `<Future pending>`.

Reply:
Ah. The documentation says _coroutines_ must be wrapped with a `Task`. `Future`s apparently don't (at least they don't mention it.) So I guess we simply:

```python
pending.add(done_fut)
```

Reply:
Next commit restores my pull-request.